package.json

```diff
@@ -8,24 +8,28 @@
   "scripts": {
     "start": "node ./bin/www",
     "devstart": "nodemon ./bin/www",
-    "serverstart": "DEBUG=express-locallibrary-tutorial:* npm run devstart"
+    "serverstart": "DEBUG=express-locallibrary-tutorial:* npm run devstart",
```

Comment:
OK good, so if I run:

```bash
npm run serverstart
npm run test
```

I get all 50 tests passing, which is nice. One thing that might be interesting is checking if Node.js built-in test runner works with these kinds of tests. I think it's been there since node 18, and it's convenient because you don't need testing frameworks as deps.